src/config/settings.js

```diff
@@ -8,13 +8,17 @@ export default {
       type: 'light', // dark also available
       primary: {
         main: '#f5f5f5',
-        light: '#ffffff',
-        dark: '#eeeeee',
       },
       secondary: {
         main: '#1967d2',
-        light: '#64b5f6',
-        dark: '#0d47a1',
+      },
+      darkened: { // custom colors used for a specific offset in some places
+        dark: '#424242',
+        light: '#eeeeee'
+      },
+      lightened: { // custom colors used for a specific offset in some places
+        dark: '#000000',
```

Comment:
It seems odd that (for the Dark theme, if I'm understanding things correctly) a "lightened" color would be `#000000` or black, while the "darkened" color would be `#424242`, which is an off-black. Should those be reversed?

I haven't yet examined the theme in detail to how or if these values are relevant so feel free to ignore this; it just seems on the surface that the "darkened" version should be darker than the "lightened" version.

Reply:
Yeah it totally could be changed.. this is really just a placeholder as my understanding was that dark theming hasn't really taken place yet.

Reply:
Sure, makes sense. If you don't mind making the update, I'd switch those two values just so we're starting with something that makes more logical sense (having the darkened variant darker than the lightened variant).

Reply:
Of course! I've made the update.

Reply:
Thanks Jack